Source/Scene/ModelExperimental/ModelExperimentalNode.js

```diff
@@ -27,32 +28,32 @@ export default function ModelExperimentalNode(options) {
   //>>includeStart('debug', pragmas.debug);
   Check.typeOf.object("options.node", options.node);
   Check.typeOf.object("options.transform", options.transform);
+  Check.typeOf.object("options.transformToRoot", options.transformToRoot);
   Check.typeOf.object("options.sceneGraph", options.sceneGraph);
   Check.typeOf.object("options.children", options.children);
   //>>includeEnd('debug');
 
   const sceneGraph = options.sceneGraph;
   const transform = options.transform;
+  const transformToRoot = options.transformToRoot;
 
   this._sceneGraph = sceneGraph;
   this._children = options.children;
   this._node = options.node;
 
   const components = sceneGraph.components;
 
+  this._transform = Matrix4.clone(transform);
+  this._transformToRoot = Matrix4.clone(transformToRoot);
+
   this._originalTransform = Matrix4.clone(transform);
   this._axisCorrectedTransform = Matrix4.clone(transform);
   ModelExperimentalUtility.correctModelMatrix(
```

Comment:
Question, is this intended to correct the matrix in place or create a new one? remember to use the new result parameter.